Description: Skip tests required non-shipped lib: ndarray_gen
 Upstream doesn't ship the required dependency.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -172,9 +172,9 @@
 name = "numeric"
 path = "tests/numeric.rs"
 
-[[test]]
-name = "oper"
-path = "tests/oper.rs"
+#[[test]]
+#name = "oper"
+#path = "tests/oper.rs"
 
 [[test]]
 name = "par_azip"
